marketplace/deployer_util/wait_for_ready.py

```diff
@@ -161,14 +185,33 @@ def is_ingress_ready(resource):
   return False
 
 
+def is_deployer_job(resource):
+  if 'app.kubernetes.io/component' in labels(resource):
+    return (labels(resource)['app.kubernetes.io/component'] ==
+            'deployer.marketplace.cloud.google.com')
+  return False
+
+
 def name(resource):
   return resource['metadata']['name']
 
 
+def labels(resource):
+  return resource['metadata']['labels']
+
+
 def total_replicas(resource):
   return resource['spec']['replicas']
 
 
+def conditions_status(type, resource):
```

Comment:
suggesting renames: `status_condition_is_true(condition_type, resource)`

(`type` is a reserved keyword in python)

Reply:
This is nice !